src/app/components/landing-page/landing-page.component.scss

```diff
@@ -0,0 +1,16 @@
+.landing__wrapper {
+  height: 100vh;
+  width: 100vw;
+  position: relative;
+}
+.mat-card-content {
+  display: flex;
+  flex-direction: column;
+}
+.mat-card {
+  border: 1px solid;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
Are you sure that this should be applied to all the mat cards across landing page?